src/gui_common/SpeciesDetailsPanelWithFossilisation.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Godot;
+
+public class SpeciesDetailsPanelWithFossilisation : VBoxContainer
```

Comment:
Could you add a summary comment on this class to mention that this is an extended version of the species details panel that adds the fossilization popup and button?